### PR TITLE
proyect Trials

### PR DESCRIPTION
**Pull request by latinogan**
review project from trials

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        <label for="Additional Courses"><input id="Additional-Courses" type="checkbox" name="improvement"> Additional Courses</label>
+        <p>Any comments or suggestions?</p>
+        <input type="text" placeholder="Enter your comment here...">
+        
+        <button id="submit" type="submit">Submit</button>
+    </form>
+</div>
+
+</body>
+</html
```
Comment (latinogan) [inline index.html:66]: - please add the closing tag to the html > An opening tag begins a section of page content, and a closing tag ends it. For example, to markup a section of text as a html, you would open the paragraph with an opening html tag <html> and close it with a closing paragraph tag </html> :muscle: :muscle:

**Review by latinogan — COMMENTED** [review]
### changes required :recycle:

- nice coding so far but there are changes required :muscle: :muscle:
  
Please check my comments below and make all required changes.
submit for a review when you are done :smile: :smile:

File: index.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	text-align: center;
+	color: red;
+	}  
+	#survey-form{
+	text-align: center;
+	align-content: center;
+	background: blue;
+	width:60% ;
+	height: 100%;
+	margin-left: 20%;
```
Comment (OLIPLICHE) [inline index.css:19]: - [ ] Please leave the comment under a line or block of code that fits the context the comment is about.
 I don't see an "unuseful space" under this block 👍.

File: index.html (hunk 1 above)
Comment (OLIPLICHE) [reply index.html:66]: ## Great! :💪

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @latinogan , 

Good job so far!
There are some issues that you still need to work on to get approved but you are almost there!

## Required Changes ♻️

- [ ] **Check the comments under the review**._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+form{
+	color: white;
+}
+#submit{
+	background-color:green ;
+	width: 80%;
+	margin-top: 6px;
+  /*  padding-top: 7px;*/
+  /* padding-left: 28px*/
```
Comment (latinogan) [inline index.css:33]: - [ ] Please do not leave unused code as a comment in your codebase. unused code makes your code not clean.:muscle: :muscle:

**Review by latinogan — COMMENTED** [review]
changes required ♻️
nice coding so far but there are changes required 💪 💪
Please check my comments below and make all required changes.
submit for a review when you are done 😄 😄

File: index.css (hunk 3 above)
Comment (phelian23) [reply index.css:33]: - [ ] Nicely done

**Review by phelian23 — APPROVED** [review]
Hi @latinogan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**